app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
```

Comment:
According to the checklist, when importing multiple classes/modules, you should use parentheses. Here's how you should do it:

python
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```
This makes your code cleaner and easier to read.